log-collector-script/linux/eks-log-collector.sh

```diff
@@ -588,6 +624,13 @@ get_docker_info() {
 # Entrypoint
 parse_options "$@"
 
-collect
-pack
-finished
+if [ -d "/.bottlerocket/" ]; then
+  echo "Detected Bottlerocket AMI"
+  is_diskfull
+  collect_logs_bottlerocket
+  finished
+else 
+  collect
+  pack
+  finished
```

Comment:
Could we create a dedicated log collector file for BR? 
We currently separate t[hem by OperatingSystem here](https://github.com/awslabs/amazon-eks-ami/tree/master/log-collector-script) so it might be helpful to create a dedicated file for BR rather than add if-else statements to the script meant for the AL2 AMI. 

Also helps limit the blast radius of anything going wrong.

Reply:
> Could you remind me why we want to add this script to this GitHub repository and not the BR repo? I think it's probably on them to provide users an easy way to collect and export logs from the instance.

We provide this script for collecting cni logs to our customers. As of now, we couldn't use this script on Bottlerocket AMI, so needed this logic.

Reply:
> Could we create a dedicated log collector file for BR? We currently separate t[hem by OperatingSystem here](https://github.com/awslabs/amazon-eks-ami/tree/master/log-collector-script) so it might be helpful to create a dedicated file for BR rather than add if-else statements to the script meant for the AL2 AMI.
> 
> Also helps limit the blast radius of anything going wrong.

Thought having a same file would make the change transparent to Customer. But agreed, having separate file would simply the if-else logic. We will also need to update our documentation so that customers know which script to run for log collection on BR AMI. Will update this branch.